src/python/pants/backend/scala/bsp/spec.py

```diff
@@ -0,0 +1,119 @@
+# Copyright 2022 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+from __future__ import annotations
+
+from dataclasses import dataclass
+from typing import Any
+
+from pants.bsp.spec import BuildTargetIdentifier, Uri
+
+
+@dataclass(frozen=True)
+class JvmBuildTarget:
+    pass
+
+
+class ScalaPlatform:
+    JVM = 1
+    JS = 2
+    Native = 3
+
+
+@dataclass(frozen=True)
+class ScalaBuildTarget:
```

Comment:
It would be good if this linked to the definition of this class in the spec.

Reply:
Will add.